packages/frontend-2/components/project/model-page/Versions.vue

```diff
@@ -119,6 +120,10 @@ graphql(`
   fragment ProjectModelPageVersionsProject on Project {
     id
     role
+    name
```

Comment:
Instead of duplicating the fields that ProjectPageHeader needs here, you should use ProjectPageHeader's `ProjectPageProjectHeader` fragment instead